about.html

```diff
@@ -0,0 +1,161 @@
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" 
+  integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" 
+  crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" 
+  integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" 
+  crossorigin="anonymous">
+  <title>About Psch Conference</title>
+</head>
+<body>
+  <main>
+    <div class="social-icons p-1">
+      <ul class="social-list d-flex justify-content-end">
+        <li class="social-list-item">
+          <a href="index.html">
+            <img src="images/facebook-socials.png" alt="Connect on Facebook">
+          </a>
+        </li>
+        <li class="social-list-item">
+          <a href="index.html">
+            <img src="images/twitter-socials.png" alt="Connect on Twitter">
+          </a>
+        </li>
+        <li class="social-list-item">
+          <a href="index.html"> English</a>
+        </li>
+        <li class="social-list-item">
+          <a href="index.html">My Page</a>
+        </li>
+        <li class="social-list-item">
+          <a href="index.html">Logout</a>
+        </li>
+      </ul>
+    </div>
+    <header>
+      <div class="topnav bg-transparent" id="myTopnav">
+        <div class="topnav-item">
+          <a href="index.html">
+            <img class="img-responsive logo" 
+            src="images/psych-logo.png" alt="Psych Conference Logo" /> 
+          </a>
+        </div>
+        <div class="topnav-links">
+          <a href="#project">Project</a>
+          <a class="active" href="about.html">About</a>
+          <a href="index.html">Home</a>
+          <a href="javascript:void(0);" class="icon menuToggle" id="menuToggle">
+          <i class="fa fa-bars fs-3"></i>
+        </a>
+        </div>       
+      </div>
+    </header>
+    <div>
+      <section class="more-info">
+        <div>
+          <h1>
+            PSYCH AND MENTAL HEALTH | SPOKEN WORDS CONFERENCE 2021
+          </h1>
+          <p class="more-info-detail"> This is an bi-annual Psychology Conference that 
+            brings togther prominent Psychologists in the world, to share knowledge,
+            tackle problems and seek solutions to impending dangers of mental health.
+            This year's conference would be in a conducive environment of Sheraton
+            Hotels, Abuja, Nigeria. It would hold for 3days, from 17th - 20th of October, 2021.
+            For this year, we'll be expecting a host of professional in the field of
+            Mental Health and there would discussion time, where we socialize and learn.
+          </p>
+        </div>
+        <div class="email-us">
+          <p>For more information on this event, <br /> please contact the helpdesk!</p>
+          <a href="mailto:nnaemekasilver@gmail.com">nnaemekasilver@gmail.com</a>
```

Comment:
- [ ] _[Optional]_ The color of the email address should be `black` and not `blue`